### ISSUE TITLE
Website says last release was in 2013

### ISSUE THREAD

**Issue by JensRantil** [jsampson/prevayler#38]
...but I am seeing commit activity here up to 2020. A release hasn't been made in...seven years? Feels like something is off.

Website: https://prevayler.org
Commits: https://github.com/prevayler/prevayler/commits/master

**Comment by jsampson** [comment]
No, that's right. Prevayler has been pretty stable for many years so there hasn't been a lot of urgency to do another release. And with higher expectations for supply chain security these days, I'm not sure we would want to be responsible for an official build anyway, with our small and unfunded maintainer group. We were getting pretty close to declaring a 2.7 release a few years back but I guess we got distracted by other things going on in the world.

We do respond to users here and on the mailing lists, so feel free to build from sources for your own needs and let us know if you run into any issues!
